src/collection/components/CollectionOrBundleOverview.tsx

```diff
@@ -302,6 +312,15 @@ const CollectionOrBundleOverview: FunctionComponent<CollectionOrBundleOverviewPr
 				);
 				return;
 			}
+
+			if (isContributor) {
+				await CollectionService.deleteContributor(
+					selectedCollectionUuid,
+					undefined,
+					user.profile?.id
```

Comment:
do we show a toast somewhere if the profile id is undefined? otherwise we should check this first and make the deleteContributor profileId type string instead of string | undefined